### PR TITLE
Add ROS 2 workspace template collection

### PR DESCRIPTION
**Pull request by BrunoB81HK**
Add ROS 2 workspace template collection

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -268,6 +268,11 @@
   contact: https://github.com/ijnek/ros-devcontainer-template/issues
   repository: https://github.com/ijnek/ros-devcontainer-template
   ociReference: ghcr.io/ijnek/ros-devcontainer-template
+- name: ROS 2 Workspace Templates
+  maintainer: Bruno-Pier Busque
+  contact: https://github.com/BrunoB81HK/ros2-workspace-devcontainer-template/issues
+  repository: https://github.com/BrunoB81HK/ros2-workspace-devcontainer-template
+  ociReference: https://ghcr.io/BrunoB81HK/ros2-workspace-devcontainer-template
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion
  ociReference: ghcr.io/brunob81hk/ros2-workspace-devcontainer-template
```

Hi 👋

The ORAS spec requires the ociReference to be in lower case. Can we update here, thanks?
The supporting tools do help convert the case, however, its better to sync as much as possible.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Thanks for adding to the collection, left one comment!

**Comment by BrunoB81HK** [comment]
Thanks for the comment! I committed your changes 😃

**Review by samruddhikhandale — APPROVED** [review]
Looks good, thanks!

**Pull request merged**